stripe.js

```diff
@@ -0,0 +1,13 @@
+var process_charge = require('./lib/stripe').process_charge;
+
+module.exports = function(request, reply) {
+ // stripe doesn't care about the result of this request, so end the request.
```

Comment:
I'd wait until process_charge has completed successfully before `reply()`-ing to Stripe. That way Stripe will repeat the webhook if we fail to respond with a 2xx response